Sources/HummingbirdCore/Utils/FlatDictionary.swift

```diff
@@ -26,104 +26,154 @@ public struct FlatDictionary<Key: Hashable, Value>: Collection, ExpressibleByDic
     public typealias Element = (key: Key, value: Value)
     public typealias Index = Array<Element>.Index
 
+    @usableFromInline
+    struct Storage {
+        @usableFromInline internal /* private */ var elements: [Element]
```

Comment:
Can we remove the commented out `/* private */` if we require everything to be `internal`